### PR TITLE
updated amend docs

### PR DESCRIPTION
**Pull request by carolinewoodhams**
- Amend simple (no reprice)
- Amend complex (with reprice)
- Index to link these two pages

I've discovered a couple of bugs, which I'll raise tickets for separately.

### CODE REVIEW

File: site/hxapi/viewamendcancel/amend_complex.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
+---
+
+---
+
+
+# Amend Booking - complex (with reprice)
+
+[API Docs](/hxapi/) > product:[amend](/hxapi/viewamendcancel/amend_complex)
+
+The booking details that can be changed for a  amend are as follows:
```
Comment (hx-emanolov) [inline site/hxapi/viewamendcancel/amend_complex.md]: Maybe it should be "... for a **complex** amend are as follows:..."?

Comment (carolinewoodhams) [reply site/hxapi/viewamendcancel/amend_complex.md]: good spot! will change that now.

File: site/hxapi/viewamendcancel/amend_simple.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+
+For a detailed explanation of the fields returned, please see the respective booking requests for [hotels](/hxapi/hotel/bkg), [parking](/hxapi/parking/bkg) and [lounges](/hxapi/lounge/bkg).
+
+## Worked Examples
+
+Below are worked examples of both the request and response for amending (simple) a booking.
+
+### UK Products Amend Booking Request (simple for a car park)
+
+The original booking was made with test details of Mr T TEST:
```
Comment (hx-emanolov) [inline site/hxapi/viewamendcancel/amend_simple.md]: Not a big deal, but it says `Mr T TEST` here, but the examples below show `<Initial>A</Initial>` and `<Surname>TESTA</Surname>`.

Comment (hx-emanolov) [reply site/hxapi/viewamendcancel/amend_simple.md]: Ah, nevermind, it's the amend... 😊

**Pull request merged**